src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/domene/GrunnlagsdataDomene.kt

```diff
@@ -89,4 +93,9 @@ data class SivilstandMedNavn(val type: Sivilstandstype,
 data class FullmaktMedNavn(val gyldigFraOgMed: LocalDate,
                            val gyldigTilOgMed: LocalDate,
                            val motpartsPersonident: String,
-                           val navn: String?)
\ No newline at end of file
+                           val navn: String?)
+
+data class TidligereVedtaksperioder(val infotrygd: FinnesTidligereVedtak)
+data class FinnesTidligereVedtak(val overgangsstønad: Boolean,
```

Comment:
Det blir litt "rart" med verb i data-klasser. For dette er jo ikke noe som kommer ut av api-et, så da burde isåfall verbet være på feltNivå - `harTidligereOvergangsstønad` ? 

`TidligereVedtakPerStønadstype`
`TidligereInnvilgetVedtak`
`InnvilgetVedtakPerStønadstype`

andre forslag?

Reply:
Hva tenker du om denne? https://github.com/navikt/familie-ef-sak/pull/975/commits/d8470a21100a135624fe203a2d741f1f02d93ea7
Det jeg lurer på er om kanskje infotrygd burde ha en egen del på rootnivå inne i `GrunnlagsdataDomene` i stedet? 

Tenker att vi alltid burde ha en ordentlig runde når vi endrer i dette objektet.

Reply:
Det kan vi godt diskutere ila dagen